components/python/sqlalchemy/Makefile

```diff
@@ -0,0 +1,60 @@
+#
```

Comment:
elixir is a plugin for sqlalchemy. elixir depends on sqlalchemy, not the other way around.